sqlmesh/core/console.py

```diff
@@ -29,9 +29,13 @@
 
 
 SNAPSHOT_CHANGE_CATEGORY_STR = {
+    None: "Unknown",
     SnapshotChangeCategory.BREAKING: "Breaking",
     SnapshotChangeCategory.NON_BREAKING: "Non-breaking",
     SnapshotChangeCategory.FORWARD_ONLY: "Forward-only",
+    SnapshotChangeCategory.INDIRECT_BREAKING: "Indirect breaking",
+    SnapshotChangeCategory.INDIRECT_NON_BREAKING: "Indirect non-breaking",
```

Comment:
what's the difference between indirect non breaking and indirect forward only

Reply:
Indirect non-breaking is reserved for future use in which the snapshot itself needs to be rebuilt but its further dependencies - not.